### PR TITLE
add multi lang with python, add husky

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: runner.js [hunk 1]
```diff
@@ -1,42 +1,59 @@
...
-  `${process.cwd()}:/usr/src/app`,
-  '-w',
-  '/usr/src/app',
-  'node:12-alpine',
-  'node',
-  `${name}.js`
-]
+const parseLanguage = (language) => {
+  return new Promise((resolve, reject) => {
+    fs.readFile('./languages.json').then(contents => {
```
Comment (stripedpajamas) [inline runner.js]: can we just import the json file at the top ? require('./languages.json')

File: languages.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "js": {
+    "extension": ".js",
```
Comment (stripedpajamas) [inline languages.json]: let's put the . in the code and make the config just js/py/etc

File: runner.js (hunk 1 above)
Comment (ghost) [reply runner.js]: fixed

File: languages.json (hunk 2 above)
Comment (ghost) [reply languages.json]: fixed

**Review by stripedpajamas — APPROVED** [review]

**Pull request merged**